index.html

```diff
@@ -5,40 +5,88 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>My Portfolio skeleton</title>
-  <link rel="stylesheet" href="style.css">
-  <link href="https://fonts.googleapis.com/css?family=Playfair+Display&display=swap" rel="stylesheet" />
-  <link href="https://fonts.googleapis.com/css?family=Poppins&display=swap" rel="stylesheet" />
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet" />
+  <title>My Portfolio mobile sk</title>
 </head>
 
 <body>
   <main>
-      <header>
-         <nav class="navbar">
-        <div class="menu-ham">
-
-            <img src="images/icon-menu.svg" alt="ham icon menu"/>
-          </div>
+    <header>
```

Comment:
- [ ] [OPTIONAL] The top bar color is not like the provided design, kindly stick to the design as much as possible

|Your design|Figma design|
|---|---|
|![Screenshot_20220526_180657](https://user-images.githubusercontent.com/25227022/170539230-01fd2d7d-0277-4cf1-aa33-fa9319e2c6b5.png)|![Screenshot_20220526_181032](https://user-images.githubusercontent.com/25227022/170539407-9ff5a7eb-0274-45ee-bbd9-dcbe7752c4d1.png)|